src/app/clusters/application-launcher-server/application-launcher-server.cpp

```diff
@@ -222,10 +387,72 @@ bool emberAfApplicationLauncherClusterHideAppRequestCallback(app::CommandHandler
     EndpointId endpoint = commandPath.mEndpointId;
     auto & application  = commandData.application;
 
+    ApplicationLauncherApplication appInput;
```

Comment:
This does raise the question: why are we not just using the Application struct from cluster-objects, instead of using the old one from af-structs (which we are trying to remove) and having to shim to it in all the command impls?  Followup for cleaning that up?